ui/app/helpers/parse-pki-cert.js

```diff
@@ -43,9 +44,13 @@ export function parseCertificate(certificateContent) {
   // field themselves are Time values.
   const expiryDate = cert?.notAfter?.value;
   const issueDate = cert?.notBefore?.value;
+  const serialNumber = Convert.ToHex(cert.serialNumber.valueBlock.valueHex)
```

Comment:
😵

Reply:
Yeah 🥴  this took way longer than I want to admit